frontend/src/metabase/setup/components/EmbeddingSetup/EmbeddingSetupSidebar.tsx

```diff
@@ -59,6 +60,7 @@ export const EmbeddingSetupSidebar = () => {
 
   return (
     <Box
+      component="aside"
```

Comment:
some of these are using proper semantic elements making testing in Cypress possible and making our components more accessible.